spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -414,7 +414,7 @@ def logger
     context 'on el-5' do
       let(:platform) { Beaker::Platform.new('el-5-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-5\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-5.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
if you're not testing the `:package_proxy` option, then you shouldn't need to provide it at all to this method.  It would give more confidence in this functionality if you didn't actually include that option, even if setting it to be false should create the same behavior.